samples/SimpleDataSource/Tables/TableBase.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Performance.SDK;
+using Microsoft.Performance.SDK.Processing;
+using System;
+using System.Collections.Generic;
+
+namespace SampleCustomDataSource.Tables
+{
+    //
+    // This is a sample base class for all regular and metadata tables in your project which helps simplify management of them.
+    // 
+    // A table is a logical collection of similar data points. 
+    // Things like CPU Usage, Thread Lifetimes, File I/O, etc. are all tables
+    //
+    // There is no explicit table interface so as to give you flexibility in how you implement your tables.
+    // All that matters is that you have some way of getting the data out of the data files and into the ITableBuilder in CreateTable   
```

Comment:
this line is misleading to me, since it implies the existence of a "CreateTable" method the SDK calls. I don't see that method, but I do see "BuildTable" on the processor interface.

I would also be explicit in mentioning that this method is inside the processor. I.e. "and into the ITableBuilder passed into the BuildTable method on your CustomDataProcessor."